### ISSUE TITLE
jquery ':focus' selector fails when window is not in focus

### ISSUE THREAD

**Issue by Bkucera** [cypress-io/cypress#2176]
The problem is jquery's ':focus' only works if the window itself is focused. 

This has been a known problem for a long time, and has a simple solution.

Here's the fix.

```js
/* replace the focus selector with our own version */
jQuery.find.selectors.filters.focus = function(elem) {
  var doc = elem.ownerDocument;
  return elem === doc.activeElement && !!(elem.type || elem.href);
}
```
seen in:
https://github.com/mattheworiordan/jquery-focus-selenium-webkit-fix/blob/master/app/assets/javascripts/jquery.focus.test-fix.js

**Comment by jennifer-shehane** [comment]
It seems to me that the work for this was done here: https://github.com/cypress-io/cypress/blob/develop/packages/driver/src/config/jquery.coffee#L49 Can you confirm if this is true @Bkucera?

**Comment by Bkucera** [comment]
fixed in #2982

**Comment by brian-mann** [comment]
If this issue was already fixed by a released version of Cypress please annotate the `Release` and ideally update the changelog to point to this.
